migaloo/assetlist.json

```diff
@@ -382,6 +382,49 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/lsdshark.png"
         }
       ]
+    },
+    {
+      "description": "Rakoff Token",
+      "denom_units": [
+          {
+              "denom": "ibc/44C29C91F202E20C8E28DFB1FA89B725C54171CD77B8948836C72E7A97E4A018",
+              "exponent": 0
+          },
+          {
+              "denom": "Rakoff",
+              "exponent": 6
+          }
+      ],
+      "base": "ibc/44C29C91F202E20C8E28DFB1FA89B725C54171CD77B8948836C72E7A97E4A018",
+      "address": "cw20:terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2",
+      "type_asset": "ics20",
+      "name": "Rakoff",
+      "display": "Rakoff",
+      "symbol": "Rakoff",
+      "traces": [
+        {
+          "type": "ibc-cw20",
+          "counterparty": {
+            "chain_name": "terra",
+            "base_denom": "cw20:terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2",
+            "port": "wasm.terra19pfxzj9580h7rjd6z6sn5x2se76vgrc0etltr79g9z3t82jzp4hq63qguc",
+            "channel_id": "channel-87"
+          },
+          "chain": {
+            "port": "transfer",
+            "channel_id": "channel-114",
+            "path": "transfer/channel-114/cw20:terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/RakoffToken/assets/main/rakoff.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/RakoffToken/assets/main/rakoff.png"
```

Comment:
The image URL must point to the Chain Registry repo. This will require uploading the image here (into the /terra/images/ directory).

Reply:
please merge https://github.com/cosmos/chain-registry/pull/4291 first so I can insert a valid URL

Reply:
please refer to https://github.com/cosmos/chain-registry/pull/4292

Reply:
The image has been uploaded, but this URL now needs to be updated (please add image_sync to chain_name: terra, base_denom: cw20:terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2)

Reply:
Updated